extensions/wikia/ThemeDesigner/ThemeDesignerController.class.php

```diff
@@ -289,24 +289,24 @@ public function executeBackgroundImageUpload() {
 	 * @param UploadBase $upload
 	 * @return array
 	 */
-	private function uploadImage($upload) {
+	private function uploadImage( $upload ) {
 		global $wgRequest, $wgUser, $wgEnableUploads;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)